Scripts/assignment/emme_assignment.py

```diff
@@ -332,7 +340,7 @@ def print_vehicle_kms(self, resultdata):
             transit_times.values(), "transit_kms.txt",
             transit_times.keys(), "time")
 
-    def calc_transit_cost(self, fares, peripheral_cost, default_cost=None):
+    def calc_transit_cost(self, transit_class, fares, peripheral_cost, default_cost=None):
```

Comment:
Parameter `transit_class` is overwritten in for-loop, so it does not have any effect.

Reply:
https://github.com/HSLdevcom/helmet-model-system/pull/183/commits/4b645a943f21dc623b5eb6f558222759a203acd2